### PR TITLE
Adding Azure Pipelines support for OSD

### PR DESCRIPTION
**Pull request by c64kernal**
This commit adds initial Azure Pipelines CI support for OpenSubdiv.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    inputs:
+      artifactName: 'osd-macOS'
+      targetPath: "  /Users/runner/OSDinst"
+
+- job: Windows
+  pool:
+    vmImage: 'VS2017-Win2016'
+  steps:
+  - script: |
+      call C:\"Program Files (x86)"\"Microsoft Visual Studio"\2017\Enterprise\VC\Auxiliary\Build\vcvars64.bat
```
Comment (thomthom) [inline azure-pipelines.yml]: Might be good to test build with static and dynamic runtimes?

Comment (davidgyu) [reply azure-pipelines.yml]: Yes, good suggestion! We'll take a look at that!

**Comment by davidgyu** [comment]
This is great! Thanks, George!

File: azure-pipelines.yml (hunk 1 above)
Comment (c64kernal) [reply azure-pipelines.yml]: +1! As David said, we can start from here and build up with lots more configurations too...

Comment (thomthom) [reply azure-pipelines.yml]: Sounds good to me. (I just recalled having issues with static CRT in the past.)

Curiously, this project is using CMake, but you are calling independent build scripts here. Any particular reason for that?

Comment (c64kernal) [reply azure-pipelines.yml]: The build script just downloads the necessary dependencies and invokes cmake -- so it's all still cmake, but the build script makes the dependency management easier and keeps the .yml files small :)

Comment (thomthom) [reply azure-pipelines.yml]: Gotcha, I didn't look too close at the script before commenting.

**Pull request merged**